### PR TITLE
screen-order-checkout Added

### PR DESCRIPTION
**Pull request by shahzaib3120**
## Ticket

PLAT-XXXX
_Related tickets:_
_Related PRs:_

## Type of PR

- [ ] Bugfix
- [ ] New feature
- [ ] Minor changes

## Changes introduced

_Describe the changes being introduced in this PR_
_Include screenshots if necessary_
_Please link any documentation reference that relate to the changes_

## Test and review

_Describe how a reviewer should test your PR_

### CODE REVIEW

**Comment by shahzaib3120** [comment]
preview

File: modules/screen-order-checkout/index.js [hunk 1]
```diff
@@ -1,26 +1,399 @@
...
+        </View>
+        <View style={orderStyles.description}>
+          <Text>Order name</Text>
+          <View style={orderStyles.bottomComponent}>
+            <View style={orderStyles.quantity}>
+              <Text>-</Text>
+              <Text
+                style={{
+                  fontWeight: "bold"
+                }}
```
Comment (saad-abid-crowdbotics) [inline modules/screen-order-checkout/index.js:276]: Move inline styling to the stylesheet.

**Review by saad-abid-crowdbotics — CHANGES_REQUESTED** [review]
Move inline styling to the stylesheet.

File: modules/screen-order-checkout/index.js (hunk 1 above)
Comment (shahraizali) [reply modules/screen-order-checkout/index.js:276]: write code for the card section as well instead of image

**Comment by shahzaib3120** [comment]
After the DebitCard component:
preview

**Review by saad-abid-crowdbotics — APPROVED** [review]

**Review by shahraizali — APPROVED** [review]

**Pull request merged**